src/current/v23.2/alter-table.md

```diff
@@ -1828,10 +1823,6 @@ To unhide the column, run:
 
 ### Alter a primary key
 
-#### Demo
-
-{% include_cached youtube.html video_id="MPx-LXY2D-c" %}
```

Comment:
oh man -1000 points to slytherin for adding YT links and then making private during a "cleanup"